script.js

```diff
@@ -0,0 +1,2 @@
+var targetElements = document.getElementsByTagName('ul');
+targetElements[0].insertAdjacentHTML('afterbegin','<li>これです</li>');
```

Comment:
IMO
If it is a tag name, there is another ui, and it will break when the order changes, so let's do id?